### PR TITLE
add(main):cbonsai

### PR DESCRIPTION
**Pull request by Anonymous1231-dev**

### CODE REVIEW

**Comment by Anonymous1231-dev** [comment]
>2022-10-09T07:48:55.0035845Z termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libvulkan.so'
2022-10-09T07:48:55.0036440Z termux-elf-cleaner: Removing the DT_AARCH64_BTI_PLT dynamic section entry from 'usr/lib/aarch64-linux-android/33/libz.so'
2022-10-09T07:48:55.1361130Z ERROR: No files in package.

WHAT?

File: packages/cbonsai/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gitlab.com/jallbrit/cbonsai
+TERMUX_PKG_DESCRIPTION="Grow bonsai trees in your terminal" 
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"                        
+TERMUX_PKG_VERSION=1.3.1
+TERMUX_PKG_SRCURL=https://gitlab.com/jallbrit/cbonsai/-/archive/v${TERMUX_PKG_VERSION}/cbonsai-v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=62aa7e0eaf3098b7a6a2787146bd2531437df6ad0e604b0f9176128797efd8f9
+TERMUX_PKG_DEPENDS="ncurses, ncurses-static, ncurses-ui-libs, ncurses-ui-libs-static, ncurses-utils"
```
Comment (Grimler91) [inline packages/cbonsai/build.sh:8]: Static packages are most likely not (runtime) dependencies

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Makefile needs to be patched to use $CC instead of hardcoded cc, and to not override PREFIX

**Comment by Anonymous1231-dev** [comment]
@Grimler91 , the build use $C instead of $CC, how?

**Comment by Anonymous1231-dev** [comment]
@Grimler91 , the build use $C instead of $CC, how?

**Comment by Grimler91** [comment]
> @Grimler91 , the build use $C instead of $CC, how?

Makefiles uses a slightly different format, so needs to be `$(CC)` instead of `$CC`

**Comment by Anonymous1231-dev** [comment]
site down... :(

**Comment by Anonymous1231-dev** [comment]
site down... :(

**Comment by Anonymous1231-dev** [comment]
Can someone ru-run the actions?

**Comment by Anonymous1231-dev** [comment]
>Makefile:2: *** Recursive variable 'CC' references itself (eventually).  Stop.

Oops

**Comment by Anonymous1231-dev** [comment]
Done!

**Comment by Anonymous1231-dev** [comment]
Can someone merge it?

**Comment by Anonymous1231-dev** [comment]
@Grimler91 ready to merge!

**Comment by Grimler91** [comment]
@Anonymous1231-dev Please re-write the commit message currently it reads:

```
New package:cbonsai

Signed-off-by: Anonymous1231-dev <87801676+Anonymous1231-dev@users.noreply.github.com>

Fix(1th attempt)

Signed-off-by: Anonymous1231-dev <87801676+Anonymous1231-dev@users.noreply.github.com>

Update deps

Signed-off-by: Anonymous1231-dev <87801676+Anonymous1231-dev@users.noreply.github.com>

Patch

Signed-off-by: Anonymous1231-dev <87801676+Anonymous1231-dev@users.noreply.github.com>

Fixed Patch

Signed-off-by: Anonymous1231-dev <87801676+Anonymous1231-dev@users.noreply.github.com>

fix

Signed-off-by: Anonymous1231-dev <87801676+Anonymous1231-dev@users.noreply.github.com>

Fix patch again

Signed-off-by: Anonymous1231-dev <87801676+Anonymous1231-dev@users.noreply.github.com>
```

which is a bit excessive. First line and (optionally) one `Signed-off-by:` would suffice

**Comment by Anonymous1231-dev** [comment]
Done!

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**